### PR TITLE
Update build.yml

### PR DESCRIPTION
**Pull request by crwaters16**
Updating build files to account for repurposing of "internet" section to "fundamentals."

### CODE REVIEW

**Comment by KimJ15** [comment]
@jrf0110 Who should we work with to approve these types of requests?

**Comment by jrf0110** [comment]
Why is #959 a separate PR from this one?

**Comment by crwaters16** [comment]
> Why is #959 a separate PR from this one?

@jrf0110- I can't figure out how to combine 958 and 959. I edited the files from Github instead of my local repo, so I didn't go through the usual process of creating a specific branch for this work. Any guidance on how to combine these appreciated!

**Comment by pedrosousa** [comment]
This PR depends on #954 which renames the `products/internet` folder to `products/fundamentals`.

**Comment by crwaters16** [comment]
> > Why is #959 a separate PR from this one?
> 
> @jrf0110- I can't figure out how to combine 958 and 959. I edited the files from Github instead of my local repo, so I didn't go through the usual process of creating a specific branch for this work. Any guidance on how to combine these appreciated!

@jrf0110 Pedro helped me figure it out, now combined into one PR and I'm closing 959!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -382,16 +382,16 @@ jobs:
           npm install
           mkdir .docs
           npm run ghactionsbootstrap
-          npm run build
+          npm run build
```
Comment (jrf0110) [inline .github/workflows/build.yml]:
```suggestion
          npm run build
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -364,15 +364,15 @@ jobs:
           mkdir .docs
           npm run ghactionsbootstrap
           npm run build
-
-  build-gateway:
+
```
Comment (jrf0110) [inline .github/workflows/build.yml]:
```suggestion

```

**Review by jrf0110 — APPROVED** [review]

**Pull request merged**